src/payment/models/buy-crypto/dto/buy-crypto-history.dto.ts

```diff
@@ -0,0 +1,12 @@
+import { AmlCheck } from '../../buy-crypto/enums/aml-check.enum';
+
+export class BuyCryptoHistoryDto {
+  inputAmount: number;
+  inputAsset: string;
+  outputAmount: number;
+  outputAsset: string;
+  txId: string;
+  date: Date;
+  amlCheck: AmlCheck;
+  isComplete: boolean;
+}
```

Comment:
All 3 history DTOs are the same. Should we use a single class/interface?

Reply:
Thought about the very same thing, decided to have it split, as I wasn't sure, if we want to have more fields for one type of DTO. Or a completely different set of information

Reply:
I think it's a good thing to have very similar history entries for different routes. It might therefore be a good idea to have multiple DTOs, but use a common base class. WDYT?

Reply:
good point, i will extract a base class and let them implement it. if changes are needed it will be easy to extend